iib/workers/tasks/utils.py

```diff
@@ -756,6 +756,9 @@ def run_cmd(
     response: subprocess.CompletedProcess = subprocess.run(cmd, **params)
 
     if strict and response.returncode != 0:
+        if set(['buildah', 'manifest', 'rm']) <= set(cmd):
+            if 'image not known' in response.stderr:
```

Comment:
Could those conditions be in one if?
`if set(['buildah', 'manifest', 'rm']) <= set(cmd) and 'image not known' in response.stderr:`

Reply:
yes, we could combine it into a single if

Reply:
Done